apps/web/src/components/Settings/ChoosePicture.tsx

```diff
@@ -0,0 +1,159 @@
+  channel: Profile
+  setNFTAvatar: (
+    contractAddress: string,
+    tokenId: string,
+    chainId: Number
+  ) => void
+}
+
+const ChoosePicture: FC<Props> = ({
+  isModalOpen,
+  onClose,
+  onChooseImage,
+  channel,
+  setNFTAvatar
+}) => {
+  const request = {
+    limit: 32,
+    chainIds: [POLYGON_CHAIN_ID, mainnet.id],
+    ownerAddress: channel.ownedBy
+  }
+
+  const { data, loading, error, fetchMore } = useProfileNfTsQuery({
+    variables: {
+      request
+    }
+  })
+
+  const collectedNFTs = data?.nfts?.items as Nft[]
+  const pageInfo = data?.nfts?.pageInfo
+
+  const { observe } = useInView({
+    rootMargin: '10px',
+    onEnter: async () => {
+      await fetchMore({
+        variables: {
+          request: {
+            ...request,
+            cursor: pageInfo?.next
+          }
+        }
+      })
+    }
+  })
+
+  return (
+    <Modal
+      onClose={() => onClose?.()}
+      show={isModalOpen}
+      panelClassName="max-w-lg h-auto"
+      autoClose={false}
+    >
+      <div className="overflow-y-auto no-scrollbar">
+        <Tab.Group>
+          <Tab.List className="flex justify-center static top-0 w-full">
+            <Tab
+              className={({ selected }) =>
+                clsx(
+                  'py-2 border-b-2 text-sm focus:outline-none w-full',
+                  selected
+                    ? 'border-indigo-500 opacity-100'
+                    : 'border-transparent opacity-50 hover:bg-indigo-500/[0.12]'
+                )
+              }
+            >
+              Upload
+            </Tab>
+            <Tab
+              className={({ selected }) =>
+                clsx(
+                  'py-2 border-b-2 text-sm focus:outline-none w-full',
+                  selected
+                    ? 'border-indigo-500 opacity-100'
+                    : 'border-transparent opacity-50 hover:bg-indigo-500/[0.12]'
+                )
+              }
+            >
+              NFT's
+            </Tab>
+          </Tab.List>
+          <Tab.Panels>
+            <Tab.Panel className="no-scrollbar focus:outline-none flex justify-center py-10">
+              <label
+                htmlFor="choosePfp"
+                className="border border-indigo-500 bg-indigo-500 px-5 py-1 rounded-full text-white cursor-pointer"
+              >
+                Choose image
+              </label>
+              <input
+                id="choosePfp"
+                type="file"
+                accept=".png, .jpg, .jpeg, .svg, .gif"
+                className="hidden w-full"
+                onChange={onChooseImage}
+              />
+            </Tab.Panel>
+            <Tab.Panel className="no-scrollbar max-h-96 overflow-y-auto focus:outline-none py-5">
+              {!error && !loading && (
+                <>
+                  <div className="grid grid-cols-3 gap-5">
+                    {collectedNFTs.map((nft: Nft) => {
+                      return (
+                        <img
```

Comment:
add fixed height for images.

<img width="539" alt="image" src="https://user-images.githubusercontent.com/29498872/211197638-4255fd70-a3a8-4c70-8443-836dedacca92.png">